docs/en/wallet-instance-attestation.rst

```diff
@@ -2,56 +2,694 @@
+
+Verifiable through the following public key (Wallet Provider Public Key)
+obtained within the Entity Configuration of the Wallet Provider,
+attested in the related Trust Chain
+
+.. code-block:: javascript
+
+  {
+  "kty": "EC",
+  "x": "qrJrj3Af_B57sbOIRrcBM7br7wOc8ynj7lHFPTeffUk",
+  "y": "1H0cWDyGgvU8w-kPKU_xycOCUNT2o0bwslIQtnPU6iM",
+  "crv": "P-256"
+  }
+
+
+Format of the Wallet Provider Entity Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Wallet Provider Entity Configuration is a JWT containing all the metadata
+relating to the Wallet Provider such as the public keys, the supported algorithms,
+the type of entity (in addition to the Wallet Provider, for example, it could also
+be a relying party) and the list of the available endpoints.
+It broadly implements openid-federation protocol.
+
+Header
+^^^^^^
++---------+----------------------------------------------------------------+
+| **key** | **value**                                                      |
++---------+----------------------------------------------------------------+
+| alg     | Algorithm to verify the token signature (es. ES256)            |
++---------+----------------------------------------------------------------+
+| kid     | Thumbprint of the public key used for signing                  |
++---------+----------------------------------------------------------------+
+| type    | Media type, in this case, we use the entity-statement+jwt value|
++---------+----------------------------------------------------------------+
+
+Payload
+^^^^^^^
++-----------------------------------+-----------------------------------+
+| **key**                           | **value**                         |
++-----------------------------------+-----------------------------------+
+| iss                               | The public url of the wallet      |
+|                                   | provider                          |
++-----------------------------------+-----------------------------------+
+| sub                               | The public url of the wallet      |
+|                                   | provider                          |
++-----------------------------------+-----------------------------------+
+| iat                               | Configuration release timestamp   |
++-----------------------------------+-----------------------------------+
+| exp                               | Configuration expiration          |
+|                                   | timestamp.                        |
```

Comment:
sometimes you add `.` at the end of the line, sometimes not

please align this accordingly by adding trailing points everywhere